resources/metadata/sonar.yaml

```diff
@@ -0,0 +1,150 @@
+metadata:
+  name: sonarExecuteScan
+  description: Executes the Sonar scanner
+  longDescription: The step executes the [sonar-scanner](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner) cli command to scan the defined sources and publish the results to a SonarQube instance.
+spec:
+  inputs:
+    params:
+      - name: changeId
+        type: string
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: changeBranch
+        type: string
+        scope:
+          - PARAMETERS
+      - name: changeTarget
+        type: string
+        scope:
+          - PARAMETERS
```

Comment:
really only as parameter?

Reply:
This was not exposed previously but added via `env.CHANGE_BRANCH` and `env.CHANGE_TARGET`. 

Or do you think we should remove the parameters completely and use `os.Getenv("CHANGE_BRANCH")`?

Reply:
I think ok for now ...